.github/CHANGELOG.md

```diff
@@ -0,0 +1,61 @@
+## Release 0.6.0a3 (development release)
+
+### New features since the last release
+
+### Bug fixes
+
+### Housekeeping changes
+
+* A large number of linting corrections were made to improve the overall pylint report. These were mostly minor but essential modifications including restructuring, re-coding, optimization, updating `.pylintrc`, adding `.coveragerc`. The code quality score is improved to `A` for the released version. Check ["Files changed"](https://github.com/XanaduAI/flamingpy/pull/11/files) for details.
+* `TODO` comments have been removed exclusively from files with low code quality grades. The Dev team has created tickets to be actioned for all removed TODO comments on separate (private) FlamingPy boards.
+* `CONTRIBUTING.md`, `code_of_conduct.md`, and `CHNAGLELOG.md` were imported and modified from the StrawberryFields project. Dev team plans to extend these with customized details in future PRs.
+
```

Comment:
Just suggesting changes to conform to PSS standards. We usually add a link to the PR where the change was added, so that it's easier to track (added this PR to all entries; not sure if that's correct).

```suggestion
### Improvements

* A large number of linting corrections were made to improve the overall pylint report. These were mostly minor but essential modifications including restructuring, re-coding, optimization, updating `.pylintrc`, adding `.coveragerc`. The code quality score is improved to `A` for the released version. Check ["Files changed"](https://github.com/XanaduAI/flamingpy/pull/11/files) for details. [(#11)](https://github.com/XanaduAI/flamingpy/pull/11)

* `TODO` comments have been removed exclusively from files with low code quality grades. The Dev team has created tickets to be actioned for all removed TODO comments on separate (private) FlamingPy boards. [(#11)](https://github.com/XanaduAI/flamingpy/pull/11)

* `CONTRIBUTING.md`, `code_of_conduct.md`, and `CHANGLELOG.md` were imported and modified from the Strawberry Fields project. Dev team plans to extend these with customized details in future PRs. [(#11)](https://github.com/XanaduAI/flamingpy/pull/11)

```